parallel-install/pkg/deployment/overrideinterceptors.go

```diff
@@ -28,6 +33,40 @@ const (
 	defaultRemoteTLSKeyEnc  = "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcEFJQkFBS0NBUUVBeG5kYktnOTluemQ4ZnVNbDllamRzTSt6eUZleW96cjAxV0FZOHFBQkUrRlR5SmZUCmREamE2UGpWc0Roa1c3dlhJWERUbjRHYmJUSzcrdFVzeFpEZmduSmQxM0lVaCtJa3JGeThwOExiV2o5WXJINUgKOEZ1b29LaE84eDJxdWlSbks4QnE5ZWo5WGlpMDh4OXBOMmE2TmFEK0RDbStLcEozMm1nK0pva3BHQnZaVUpRSwp6RElHSDdtQWxZU3lNWGVJUWo4cGxwU1JXMWtKTHJpdktBVlAxVHB4NlBQN3JyQUZqL1hGRktZY3pac0FCNWkxCmtlVTZWbnBZa0ltYXdkNVJJaUNWMFN0MkozamUxQUlKeUJKWmhBdC9mSzdkb1Fqd3hKN1o0WUdZdXJsQzEwNlQKbWVxNlVPcXAwZDB2dC9tZ1ppTkVKbFZrRlhVRXVoR2x1WTI0UVFJREFRQUJBb0lCQVFDSjhndjdnQ2pnc2NCbQpzWnVCQVFxV0NzZjdTSGx4MjFpeHRzbWdXblpsU3dqaE5DWlZjZTgyWHo2bjdZcFQrSXZmUW56Vk1WREc1YXlpCisralNxWSt4SzZ6dVF1emlSZDBYc0oyd1BWQVp1azM0Rnc0Smtxdnlmd25oRVkzSk0rUkNGTXhEZ0Y0YlJGQUIKYktQRlRqRy9kTmNmdlNQZ2swMmJFVG1ocjFSUTNFL2F1bURYMS9mVldkVzduMjZndVpkdjlhR0NQMy9QMThtNApFMXlUMVRaYVdUcVVSQ09sU2J5WUh0R2xvdFdtVm14b3A5SktVMVhoVk11bVhBUnZjRUN5blpBY3g0L1cvKzV6Ck81Zld4cEZtZ0QwOEtUUEdoN0R0QjB0eXlkTlZXZXBjM0VZOSs0dFVXamNSTXRKRVpST29mWGNIelV0OFN4c2EKZU1zQmdDUUJBb0dCQU0zRXAzWS9vbi9IU0ovUjlpRUpQNE95SjlnTEw0UXEzR01UcnUzSFBBRDlrY3g3STg5YwpqeU8yMnorNEFJL3RzZFo3TDFubHJrSVlNR3FYYlByQXBTek1oQXBFWUl5cG10aEVneWNTZE9yYmFMaTVOQmpLCnBNMDBhaXgyZW5SeEFwOFEyMkkrTml0Wm1jYmFhTkhERHdIaDdBVnpGVnN0dmJZN0xEcEp2bkRaQW9HQkFQYnEKWHltSVgvV1hNZlpRVjJPUk1zR3kvd2czVUluU05YZ1JWOTU0c2Q4cVRBQnc5N0Z3TFJuRWNxMUMwSTdrS0xoQgpaU2tKUDk3Y0lINXBHNnBIOXBuWHZBQmpCdWc5cW8rclozSEI4Vjd3ZU5waTNzSnBRVFZYUFo4bmVBdStuN1pDCnJsZWJMYUNYUUViN2lRZHA4elRJLzJoa291MXRCM2tXQjBuV3kyR3BBb0dCQU1CdVBaSFhWdmVhZmUrQW9tWW8KeU80M2FRMmhBRkhnNTNQOGoyWXRJWTluazdjZ0hkQXBwbTltN1VsOG9ZSDRiNHkrYlB6c1QvZmR1VUdsMVRQMwpmMEVURGhTdjkzNzBpaXZnZnFyR2x2S2dPQ0l3aVdqNThmODZHbVQwYy9aN1RWRkdxWFFKN0F6RVlZeFc2eG5vCkNodmZsU05QaWRSWVJZZXJkT1FaM1BDWkFvR0FSY3BoTTRBVWYzcEk2UEkwZ1RRZFFKcXpjME1QUktWaDc1b1gKV0E2TldDTEFjSzk5azIyOWtiYnhJdi9ycXpmYU9wcGhXWVAveGFJNm5RQmdqWFRod3dJelpYaVlEelMrN1BUcAp2RUd4VThCc3FHMmh3Um0zRUxpajlrUlZyaHduVUlEd2ZscWlQdTRCZ1E2LzRKU1Y1YW1hWjR0cWNlbUxYekpXCnhRd3RXR0VDZ1lBbWUvM0c1RXN4T0ZkUmltcTRwYjlMMjJzVnFqTFFVcUw1R09QVWNBYlZERDJ3SEU0K0pQdEsKUXdPTVJBZ05oeEJPMmRxWGk0SXgxWmRuRmE5dENFTmNrQzFRL2VmZzUwN2dhdk44ZGJGaGQ1dGpBTU9zVGVDOQp6SnowUFc2VktOMkptd0JWMlhIeWlDTHJ2ZjlnZVVtSHhzR3A5dEhkRlc5S2g1allXNjRRMEE9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo="
 )
 
+var retryOptions = []retry.Option{
+	retry.Delay(2 * time.Second),
+	retry.Attempts(3),
+	retry.DelayType(retry.FixedDelay),
+}
+
+var coreDNSPatchTemplate = `
```

Comment:
please move coreDNS related things to a separate file in the same package. Then it's nicely separated from other things here and can be moved easily if necessary.

Reply:
Moved to separate file.